### ISSUE TITLE
Improve Accessibility of Components - Semantic HTML, et c

### ISSUE THREAD

**Issue by livoszlak** [livoszlak/book-finder-api-2.0#3]
Review and improve the accessibility of the components, ensuring they meet WCAG guidelines.

This issue's scope should be focused on making sure components render with appropriate semantic HTML attributes. As for styling and colors, there is a separate issue for that here: #4

**Comment by Sarveshk76** [comment]
Hi @livoszlak , can I work on this issue?

**Comment by livoszlak** [comment]
> Hi @livoszlak , can I work on this issue?

Sure thing! 💐

**Comment by Prince-Alarming** [comment]
Are you still working on this? Would like to help if so.
